source/feathers/controls/text/BitmapFontTextRenderer.as

```diff
@@ -1075,7 +1075,7 @@ package feathers.controls.text
 			this._characterBatch.x = this._batchX;
 
 			result.x = maxX;
-			result.y = currentY + lineHeight;
+			result.y = currentY + lineHeight - this.currentTextFormat.leading;
```

Comment:
This way leading functions like the gap property in layouts: It adds the extra space just in between lines and does not increase the height of single line text.